modules/gdscript/gdscript_parser.cpp

```diff
@@ -1095,8 +1095,36 @@ GDScriptParser::VariableNode *GDScriptParser::parse_variable(bool p_is_static, b
 	}
 
 	complete_extents(variable);
+
+	bool malformed_setget_found = false;
+
+	if (p_allow_property && match(GDScriptTokenizer::Token::IDENTIFIER)) {
+		StringName identifier_name = previous.get_identifier();
```

Comment:
I think `check()` and `current` should be used here instead of `match()` and `previous` since `match()` advances the parser.